### PR TITLE
FEATURE: implement checkbox label in workspace module

### PR DESCRIPTION
**Pull request by crydotsnake**
**Review instructions**

Currently, in the Workspace module, in the overview, we have a checkbox for selecting all changes. But this checkbox is not marked with a label. For clarity (and looks), it is better if it is clearly marked what this checkbox does.

### Before
neos-backend-workspace-module-checkbox-without-label

### After
neos-backend-workspace-module-checkbox-with-label

**Checklist**

- [x] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [x] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [x] Reviewer - The first section explains the change briefly for change-logs
- [x] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

**Comment by crydotsnake** [comment]
A small side question: How are the other translations generated? Does that webplate automatically?

**Review by mhsdesign — APPROVED** [review]
... but i have no clue how webplate works at all ^^

File: Neos.Neos/Resources/Private/Translations/de/Modules.xlf [hunk 1]
```diff
@@ -265,6 +265,10 @@
         <source>All changes from workspace "{0}" have been discarded.</source>
         <target state="final">Alle Änderungen aus dem Arbeitsbereich "{0}" wurden verworfen.</target>
       </trans-unit>
+      <trans-unit id="workspaces.checkbox.label" approved="yes">
```
Comment (kdambekalns) [inline Neos.Neos/Resources/Private/Translations/de/Modules.xlf]:
```suggestion
      <trans-unit id="workspaces.selectAllCurrentChanges" approved="yes">
```

File: Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
 										<input type="checkbox" id="check-all" /><span></span>
 									</label>
 								</th>
-								<th></th>
+								<th>{neos:backend.translate(id: 'workspaces.checkbox.label', source: 'Modules', package: 'Neos.Neos')}</th>
```
Comment (kdambekalns) [inline Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html]: That is a very generic, err, label. ;) It's not as if there is only one checkbox in the workspace module. May I suggest this:

```suggestion
								<th>{neos:backend.translate(id: 'workspaces.selectAllCurrentChanges', source: 'Modules', package: 'Neos.Neos')}</th>
```

File: Neos.Neos/Resources/Private/Translations/en/Modules.xlf [hunk 3]
```diff
@@ -199,6 +199,9 @@
 			<trans-unit id="workspaces.allChangesInWorkspaceHaveBeenDiscarded">
 				<source>All changes from workspace "{0}" have been discarded.</source>
 			</trans-unit>
+			<trans-unit id="workspaces.checkbox.label">
```
Comment (kdambekalns) [inline Neos.Neos/Resources/Private/Translations/en/Modules.xlf]:
```suggestion
			<trans-unit id="workspaces.selectAllCurrentChanges">
```

**Review by kdambekalns — CHANGES_REQUESTED** [review]
I'd rather use a different id for the new translation.

Weblate will just do the right thing, any XLIFF file will get the label and translation as needed.

**Review by mhsdesign — APPROVED** [review]

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**